lib/workers/repository/onboarding/branch/create.ts

```diff
@@ -25,7 +26,16 @@ export async function createOnboardingBranch(
     config,
     configFile!,
   );
-  const commitMessage = commitMessageFactory.create();
+  let commitMessage = commitMessageFactory.create().toString();
+
+  if (config.commitBody) {
+    commitMessage = `${commitMessage}\n\n${template.compile(
+      config.commitBody,
+      config,
```

Comment:
Directly use `gitAuthor` value instead of `template.compile`, as we do not want other template values.
Ref: https://github.com/renovatebot/renovate/issues/25943#issuecomment-1869494422

Reply:
yes